Project_Organization.md

```diff
@@ -51,16 +51,27 @@ to participate in decisions that require a Supermajority Vote.
 ### 3.1. Steering Committee
 
 The Steering Committee is the body that is responsible for governing the
-Project, coordinating Working Groups, and managing assets.
+Project, coordinating Working Groups, and managing assets. To keep discussions 
+productive and balanced, attendance is generally limited to two attendees per 
+organization. The Steering Committee may grant an exception to this rule for a 
+specific meeting and purpose only.
 
 ### 3.2. Chairperson
 
 The Project will have a Chairperson whose responsibilities include leading the
 Steering Committee, organizing meetings, organizing voting, identifying new
 Working Groups, managing finance, tracking Working Group progress, ensuring
```

Comment:
nit: I would put "managing finance" after "organizing voting", since it is a bit odd to have it in the middle of duties related to working groups.